### PR TITLE
Quiz Correct Answer Explanation

### PR DESCRIPTION
**Pull request by heathermoses**
## Testing Instructions
1. For each lab, complete the quiz.
2. Review the correct answers and the associated explanation
3. Test that all of the explanations are there, and test that the explanation makes sense

- [x ] No discrepancies across browsers. Eg. chrome vs safari
- [x ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ x] CSS (https://jigsaw.w3.org/css-validator/)
- [x ] Google sign-in is operational
- [ x] Accessibility functions
- [x ] Pages can scale without distorting page
- [ x] No dead links
- [x ] Navbar is consistent across the site
- [x ] Pages are screen-reader accessible
- [x ] Contrast meets standards for accessibility
- [x ] Pages are keyboard accessible
- [ x] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

**Comment by notjoncruz** [comment]
For those that include data, I wonder if we should reference the source. Thoughts?

**Review by notjoncruz — APPROVED** [review]

**Comment by heathermoses** [comment]
> For those that include data, I wonder if we should reference the source. Thoughts?

Great idea, I can add citations!

File: server/database/schema.sql [hunk 1]
```diff
@@ -3057,7 +3132,8 @@ to test your knowledge. Click "Next" to start!', e'{
       {
         "val": 1,
         "type": "1",
-        "content": "8th grade"
+        "content": "8th grade",
+        "explanation": "To reach a broader audience, an 8th grade reading level is recommended (https://pmc.ncbi.nlm.nih.gov/articles/PMC8328867/)."
```
Comment (notjoncruz) [inline server/database/schema.sql]: Should the source be in its own entry, like `source`?

Comment (heathermoses) [reply server/database/schema.sql]: I can change that

**Review by notjoncruz — APPROVED** [review]
LGTM, thanks Heather

**Comment by mjd3502** [comment]
Looks great, correct answers with explanation/sources appear when reviewing quiz answers!

**Pull request merged**